types/hippy__react/index.d.ts

```diff
@@ -935,7 +940,7 @@ interface TextProps extends LayoutableProps, ClickableProps, TouchableProps {
      * The default is `tail`.
      */
     ellipsizeMode?: 'head' | 'middle' | 'tail' | 'clip';
-    children?: number | string | string[];
+    children?: number | string | string[] | React.ReactNode;
```

Comment:
`ReactNode` can be `string | number` based on the source file
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L232-L240
```suggestion
    children?: string[] | React.ReactNode;
```